playwright-action/.github/workflows/playwright.yml

```diff
@@ -0,0 +1,27 @@
+name: Playwright Tests
```

Comment:
This is already present in master: https://github.com/layer5labs/meshmap-snapshot/blob/master/.github/workflows/playwright.yml

Reply:
yh i observed that after i pushed, my master branch wasnt up to-date